taxi/views.py

```diff
@@ -1,3 +1,15 @@
 from django.shortcuts import render
 
-# Create your views here.
+from .models import Driver, Manufacturer, Car
+
+
+def index(request):
+    return render(
+        request,
+        "taxi/index.html",
+        context={
+            "num_drivers": Driver.objects.all().count(),
+            "num_manufacturers": Manufacturer.objects.all().count(),
+            "num_cars": Car.objects.all().count()
```

Comment:
`all()` methods is still here ((
there is no need to use all method before count method